TestModels/HistoryAfter/test/SimpleStringImplTest.dfy

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+include "../src/Index.dfy"
+
+module  SimpleStringImplTest {
+    import SimpleString
+    import opened SimpleTypesSmithyStringTypes
+    import opened Wrappers
+    import opened Histories
+    method{:test} GetString(){
+        var client :- expect SimpleString.SimpleString();
+        TestSingleCall(client);
+        TestMultipleCallsToSameOperation(client);
+        TestMultipleCallsToDifferentOperations(client);
+    }
+
+    method TestSingleCall(client: ISimpleTypesStringClient)
+      requires client.ValidState()
+      modifies client.Modifies
+      ensures client.ValidState()
+    {
+        var history := new History();
```

Comment:
Is this not a ghost?

Reply:
It's automatically inferred to be (since the History constructor is ghost if nothing else), so not necessary to explicitly add the `ghost` keyword, but I will just to be clearer.